### ISSUE TITLE
Jurassic World Evolution 2: Frame Generation greyed out

### ISSUE THREAD

**Issue by vlbastos** [Nukem9/dlssg-to-fsr3#80]
As the title says. I disabled v-sync, enabled and disabled, no go.
Windows scheduler is on.
![image](https://github.com/Nukem9/dlssg-to-fsr3/assets/50724237/e06d43ed-0e75-49d0-af77-460cdf948bf7)

**Comment by vlbastos** [comment]
Tried with DLSSTweaks, no luck either.

**Comment by MJ12MJ** [comment]
Jurassic World Evolution 2 uses a proprietary engine, most likely the problem is due to that. Have you checked that there are no more .exe executable files in the game folder?

**Comment by vlbastos** [comment]
> Jurassic World Evolution 2 uses a proprietary engine, most likely the problem is due to that. Have you checked that there are no more .exe executable files in the game folder? I can't verify because I don't own the game, but the advice I can give you is to verify that you have inserted the files in the primary executable of the game, they are usually found in folders named Win64 or simply BIN.

Yes, I checked all folders. I tried with version 0.60 too, which worked for me in A Plague Tale Requiem, but it's still unavailable.

**Comment by wykydtron253** [comment]
Hi there, I'm seeing this as well even with .81 release. Id be happy to steam gift the game to a dev who can help with this!
